schemas/README.md

```diff
@@ -825,6 +1157,137 @@ Intensity of detections
 </tr>
 </table>
 
+## LinePrimitive
+
+A primitive representing a series of points connected by lines
+
+<table>
+  <tr>
+    <th>field</th>
+    <th>type</th>
+    <th>description</th>
+  </tr>
+<tr>
+<td><code>type</code></td>
+<td>
+
+[enum LineType](#enum-linetype)
+
+</td>
+<td>
+
+Drawing primitive to use for lines
+
+</td>
+</tr>
+<tr>
+<td><code>pose</code></td>
+<td>
+
+[Pose](#pose)
+
+</td>
+<td>
+
+Origin of lines relative to reference frame
+
+</td>
+</tr>
+<tr>
+<td><code>thickness</code></td>
+<td>
+
+float64
+
+</td>
+<td>
+
+Line thickness
+
+</td>
+</tr>
+<tr>
+<td><code>scale_invariant</code></td>
+<td>
+
+boolean
+
+</td>
+<td>
+
+Indicates whether `thickness` is a fixed size in screen pixels (true), or specified in world coordinates and scales with distance from the camera (false)
+
+</td>
+</tr>
+<tr>
+<td><code>points</code></td>
+<td>
+
+[Point3](#point3)[]
+
+</td>
+<td>
+
+Points along the line
+
+</td>
+</tr>
+<tr>
+<td><code>color</code></td>
+<td>
+
+[Color](#color)
+
+</td>
+<td>
+
+Solid color to use for the whole line. One of `color` or `colors` must be provided.
+
+</td>
+</tr>
+<tr>
+<td><code>colors</code></td>
+<td>
+
+[Color](#color)[]
+
+</td>
+<td>
+
+Per-point colors (if specified, must have the same length as `points`). One of `color` or `colors` must be provided.
+
+</td>
+</tr>
+<tr>
+<td><code>indices</code></td>
+<td>
+
+uint32[]
+
+</td>
+<td>
+
+Indices into the `points` and `colors` attribute arrays, which can be used to avoid duplicating attribute data.
```

Comment:
ELI5?

Reply:
The diagram here is pretty good: http://www.opengl-tutorial.org/intermediate-tutorials/tutorial-9-vbo-indexing/


![](http://www.opengl-tutorial.org/assets/images/tuto-9-vbo-indexing/indexing1.png)

Reply:
this is over my head so I'm just gonna assume you know what you are talking about 😅